.eslintrc

```diff
@@ -19,7 +24,8 @@
         "selector": "ExpressionStatement[expression.callee.object.name='userEvent']",
         "message": "Await the promised result of a userEvent interaction"
       }
-    ]
+    ],
+    "react/react-in-jsx-scope": "off"
```

Comment:
I wonder if we actually need this, or if we should do what's documented in the rule for our use-case of using the automatic JSX transform:

>If you are using the [new JSX transform from React 17](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports), you should disable this rule by extending [react/jsx-runtime](https://github.com/jsx-eslint/eslint-plugin-react/blob/8cf47a8ac2242ee00ea36eac4b6ae51956ba4411/index.js#L165-L179) in your eslint config (add `"plugin:react/jsx-runtime"` to `"extends"`).

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md#when-not-to-use-it

Reply:
I feel a mix of 🤦 for missing this and 🚀 that it's better after [this commit](https://github.com/18F/identity-idp/pull/9706/commits/ef7098bc9fd9321deaabf071b5bd0ab89225250c). Thanks!